language-support/java/codegen/src/test/scala/com/digitalasset/daml/lf/codegen/CodeGenRunnerTests.scala

```diff
@@ -37,7 +37,7 @@ class CodeGenRunnerTests extends AnyFlatSpec with Matchers with BazelRunfiles {
 
     val (interfaces, pkgPrefixes) = CodeGenRunner.collectDamlLfInterfaces(conf)
 
-    assert(interfaces.length == 19)
+    assert(interfaces.length == 20)
```

Comment:
@sofiafaro-da Can you confirm there is one more package in archive 1.11 than in archive 1.8 ?

Reply:
Yes, the stable package for DA.Set.Types

Reply:
:+1:

Reply:
Would it make sense and be feasible to compare the expected names instead of just checking whether the count checks out? In this way this question would answer itself in the test. 🙂

Reply:
It is probably possible. let's ask application-runtime team ? ;)

Reply:
😂 

Good point, I'll make a note to have a look at this once this has been merged.

Reply:
Would [this](https://github.com/digital-asset/daml/pull/8921) make sense?

Reply:
If you think it is better, I have no objection for your proposal.

Reply:
I have a double though about this. 
As long it is hard-coded I will prefer to not be changed. 
This will just make my life more complicated each time I have to update the compiler version. 
The test should be designed to work with any version of LF.

Reply:
Please take into account we have a lot of test to fix every time we roll the LF version.

Reply:
Let's keep the discussion in that thread, sorry for opening it up here.

Reply:
What the test is verifying actually. Is it not enough to check it contains "Foo:Foo" ?